sections/rich-text.liquid

```diff
@@ -14,7 +14,7 @@
           <div class="rich-text__text rte" {{ block.shopify_attributes }}>{{ block.settings.text }}</div>
         {%- when 'button' -%}
           <a{% if block.settings.button_link == blank %} aria-disabled="true"{% else %} href="{{ block.settings.button_link }}"{% endif %} class="button{% if block.settings.button_style_secondary %} button--secondary{% else %} button--primary{% endif %}" {{ block.shopify_attributes }}>
-            {{ block.settings.button_label | escape }}
+            {{ block.settings.text | escape }}
```

Comment:
In this case I don't think the update is required as there can only ever be one `button` 
<img width="366" alt="Screen Shot 2021-08-11 at 8 41 41 AM" src="https://user-images.githubusercontent.com/22102095/129030461-eefb23d3-b2ad-458f-a95c-1a75e5bc8eed.png">